### PR TITLE
[RATOM-204] Adds endpoint and custom response.

### PR DESCRIPTION
**Pull request by JeremyGibson**
Here is what seems to be the basic model for extracting the id's from an elasticsearch query.

This also adds a management command that allows a file to be added to one of the existing sample data accounts.  I added this as we needed a reproducible set of queries that would return more than one file.  This one `localhost:8000/api/v1/export/?account=6&search_simple_query_string=checke&` Should give you a file with two nodes from albert meyers and one from bill rapp.

This is WIP since we have to link up front and back and there may be more work to do with the response.  Also, tests need to be added once this is firmed up.

### CODE REVIEW

File: api/serializers.py [hunk 1]
```diff
@@ -172,6 +172,20 @@ class Meta:
         ]
 
 
+class ExportDocumentSerializer(serializers.Serializer):
```
Comment (copelco) [inline api/serializers.py]: I think ``ExportDocumentSerializer`` can be removed now

File: api/views/export.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    media_type = "application/x-gzip"
+    format = "gzip"
+    charset = None
+    render_style = "binary"
+
+    def render(self, data, accepted_media_type=None, renderer_context=None):
+        return data
+
+
+def decompose_drf_data(od):
```
Comment (copelco) [inline api/views/export.py]: ``decompose_drf_data`` can be removed

File: api/sample_data/data.py [hunk 3]
```diff
@@ -49,3 +45,34 @@ def reset_dataset(title, files, source, spacy_model):
         ratom_file.reported_total_messages = ratom_file.message_set.count()
         ratom_file.import_status = File.COMPLETE
         ratom_file.save()
+
+
+def sample_reset_all():
```
Comment (copelco) [inline api/sample_data/data.py:50]: Should we just add a 2nd "file" for Bill Rapp to the sample data? I think having a single "reset sample data" command just seems simpler and will be less confusing in the future.

File: api/tests/search/test_search.py [hunk 4]
```diff
@@ -50,3 +52,31 @@ def test_label_match__or(url, api_client, sally1, sally2, sally3, event, org, da
     matched_messages = [result["id"] for result in response.data["results"]]
     assert sally1.pk in matched_messages and sally2.pk in matched_messages
     assert sally3.pk not in matched_messages
+
+
+def test_export_match_one_file(export_url, api_client, sally1, eric1, org, event):
```
Comment (copelco) [inline api/tests/search/test_search.py:57]: 💯

File: api/tests/search/test_search.py [hunk 5]
```diff
@@ -50,3 +52,31 @@ def test_label_match__or(url, api_client, sally1, sally2, sally3, event, org, da
...
+    sally1.audit.labels.add(event)
+    sally1.save()
+    eric1.audit.labels.add(org)
+    eric1.save()
+    response = api_client.get(
+        export_url, data={"labels_importer__terms": f"{org.name}"}
+    )
+    resp_data = json.loads(gzip.decompress(response.data))
+    assert len(resp_data) == 1
+    assert eric1.file.filename in list(resp_data.keys())
```
Comment (copelco) [inline api/tests/search/test_search.py:67]: I think verifying the source_id is in the data would be a nice addition

**Review by copelco — CHANGES_REQUESTED** [review]
Looks good! Made a few suggestions.

File: api/sample_data/data.py (hunk 3 above)
Comment (JeremyGibson) [reply api/sample_data/data.py:50]: I dunno. There might be cases where you don't want Albert Meyers mixed in with Bill Rapp.  I thought this way the person could choose, or we could have three [Sample Data] accounts one that has both combined.

Comment (copelco) [reply api/sample_data/data.py:50]: Okay seems fine!

**Review by copelco — APPROVED** [review]
Once ExportDocumentSerializer is removed, we're good to merge :shipit:

**Pull request merged**